nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_dataset.py

```diff
@@ -41,9 +41,11 @@ def __init__(
         separate_prompt_and_response_with_newline: bool = False,
         answer_only_loss: bool = True,
         truncation_field: str = "answer",
-        pad_to_max_length: bool = True,
+        pad_to_max_length: bool = False,  # (@adithyare) allows for much faster training especially in PEFT settings.
```

Comment:
don't change the default for backward compatibility just in case others rely on the default behavior

Reply:
fixing..